pkg/install/v1alpha1/jiva_snapshot.go

```diff
@@ -31,18 +31,34 @@ spec:
   taskNamespace: {{env "OPENEBS_NAMESPACE"}}
   run:
     tasks:
-    - jiva-volume-podsinopenebsns-default
+    - jiva-snapshot-podsinopenebsns-default
     - jiva-snapshot-create-listsourcetargetservice-default
     - jiva-snapshot-create-invokehttp-default
   output: jiva-snapshot-create-output-default
 ---
 apiVersion: openebs.io/v1alpha1
 kind: RunTask
+metadata:
+  name: jiva-snapshot-podsinopenebsns-default
```

Comment:
Do we intend to use it as a common runtask?
Why is the name of this runtask different than others?

Reply:
Yes ... that was the mistake , we were using `.Volume` property for snapshot related operations as well. and that was breaking the runtask .

Reply:
the snapshot and the volume requests are using different cast engines and the values are passed in .Snapshot and .Volume objects.

Reply:
can this RunTask be named with `jiva-snapshot-create-`